doc/user-guide/core-functionalities/fields/fields-on-grid.cc

```diff
@@ -12,7 +12,7 @@ using atlas::array::make_shape;
 using atlas::array::make_view;
 
 int main( int argc, char* argv[] ) {
-    atlas::Library::initialise( argc, argv );
+    atlas::Library::instance().initialise( argc, argv );
```

Comment:
Sorry about this issue sneaking in. My idea was to simplify the API as follows.
I was intending to have instead use:
```c++
#include "atlas/library.h"       // will include correct "atlas/library/Library.h"
<snip>
atlas::initialise(argc,argv);    // delegates to atlas::Library::instance().initialise(argc,argv) ;
<snip>
atlas::finalise();               // delegates to atlas::Library::instance().finalise() ;
```
(see https://github.com/ecmwf/atlas/blob/develop/src/atlas/library/Library.h#L30-L31)

Do you mind making these changes in all the other code examples as well? 😇

Reply:
@wdeconinck Thank you for the suggestion.  All other code examples have been changed to reflect the cleaner API.